release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -453,6 +482,28 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-secondary-device-metrics-network-attachments"]
+==== Secondary device metrics can be associated with network attachments
+
+Secondary devices, or interfaces, are used for different purposes. It is important to have a way to classify them so that you can aggregate the metrics for secondary devices with the same classification.
+
+The kubelet is already publishing a set of network observable related metrics. The labels in these metrics contain, among others:
+
+* Pod name
+* Pod namespace
+* Interface name, such as eth0
+
+This works well until new interfaces are added to the Pod, for example via Multus, as it will not be clear what the interface names refer to. The interface label refers to the interface name, but it is not clear what that interface is meant for. In case of many different interfaces, it would be impossible to understand what network the metrics we are monitoring refer to. This is addressed by introducing the new `pod_network_name_info` metric, which can be used to build queries containing both the values exposed by the kubelet and the name of the network attachment definition the metrics relates to, which identifies the type of network.
+
+[id="ocp-4-6-cnf-tests-discovery-mode"]
+==== CNF tests can be run in discovery mode
+
+There is an optional mode where the Cloud-native Network Functions (CNF) tests try to look for configurations on the cluster instead of applying the new ones. The CNF tests image is a containerized version of the CNF conformance test suite. It is intended to be run against a CNF-enabled {product-title} cluster where all the components required for running CNF workloads are installed.
+
+The tests must perform an environment configuration every time they are executed. This involves items such as creating SRIOV Node Policies, Performance Profiles, or PtpProfiles. Allowing the tests to configure an already configured cluster might affect the functionality of the cluster. Also, changes to configuration items such as SRIOV Node Policy might result in the environment being temporarily unavailable until the configuration change is processed.
```

Comment:
We have this documented elsewhere as [SR-IOV](https://docs.openshift.com/container-platform/4.5/virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.html#discover-sr-iov-devices_virt-configuring-sriov-device-for-vms)

Reply:
Great catch!